ansible/playbooks/roles/kubernetes_master/tasks/configure-logrotate.yml

```
@@ -0,0 +1,9 @@
+---
+- name: Copy logrotate config
+  template:
+    dest: /etc/logrotate.d/k8s-apiserver
+    src: apiserver-logrotate.conf.j2
```

```suggestion
    dest: /etc/logrotate.d/k8s-apiserver-audit
    src: apiserver-audit-logrotate.conf.j2
```

Not sure if we need logrotate, please check: https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#log-backend

---

I see, good catch

---

This part was removed